core/domain/feedback_validators.py

```diff
@@ -76,16 +71,12 @@ def _get_external_id_relationships(cls, item):
                     '%s_ids' % item.entity_type,
                     prod_validators.TARGET_TYPE_TO_TARGET_MODEL[
                         item.entity_type], [item.entity_id]))
-        if (
-                item.last_nonempty_message_author_id and
-                utils.is_user_id_valid(
-                    item.last_nonempty_message_author_id)
-        ):
+        if item.last_nonempty_message_author_id:
             field_name_to_external_model_references.append(
                 base_model_validators.UserSettingsModelFetcherDetails(
                     'last_nonempty_message_author_ids',
                     [item.last_nonempty_message_author_id],
-                    may_contain_system_ids=False,
+                    may_contain_system_ids=True,
```

Comment:
Why is this changed? I didn't think that system IDs could create feedback threads.